R/knitr-engine.R

```diff
@@ -20,12 +20,12 @@
 #' @export
 eng_python <- function(options) {
   
-  ast <- import("ast")
+  ast <- import("ast", convert = TRUE)
```

Comment:
convert = TRUE is the default, why include it explicitly? (actually don't have any problem with you doing so as it provides documentation of intent, just curious)

Reply:
Right; I just did that to be explicit here.